migrate-mongo-config.js

```diff
@@ -0,0 +1,39 @@
+// Not sure if this actually works properly but it is required to run the 'migrate-mongo create [description]' command
```

Comment:
I think this file can be removed now, right?

Reply:
Yes! Removed it and added it to .gitignore in the latest commits 98fa776 9f084f5 608acdf